.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```

Comment:
Might wanna add `"semi": true` so we match the prettier config from the other boilerplate